packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -58,7 +58,7 @@
         <tr v-if="showShareDate" data-testid="shared-date">
           <th scope="col" class="oc-pr-s" v-text="shareDateLabel" />
           <td>
-            <span v-oc-tooltip="shareDateTooltip" v-text="displayShareDate" />
+            <span v-oc-tooltip="shareDateTooltip" tabindex="0" v-text="displayShareDate" />
```

Comment:
Same issue with the `tabindex` as in https://github.com/owncloud/owncloud-design-system/pull/1787#discussion_r755966806